### PR TITLE
feat: add stats page

### PR DESCRIPTION
**Pull request by cmunns**
## Remaining tasks
- [ ] There is an abrupt skip / jank when the marquee runs out of text. See video below. (also, it consistently happens upon page load...marque runs, but less than second later, marque starts again)
- [x] Fix display of "3 arrow" image at top of stat card so that there is no white strip on Safari
- [x] Fix capitalization of marquee text.  Should be: "NFT.Storage"
- [x] Fix case of the text between square brackets ("Week over week change" and "Total deals"), 
- [x] The number of total deals should probably have a comma (e.g., 34,959) or use toLocaleString.
- [x] Capitalization of logo images alt tags need to change from "Nft.Storage Users" to "NFT.Storage Users"

- [x] resolve issue with Marketplace logos broken on stats page and also on home page (may be cloudflare caching issue again)

also refactor logos component out of index.js, update footer with new link and space for heart icon

### CODE REVIEW

**Comment by dashcraft** [comment]
Using live data:
image

**Review by Codigo-Fuentes — CHANGES_REQUESTED** [review]
**todo (blocking):** Fix casing of marquee text.

Should be: "NFT.Storage"

case

**Comment by Codigo-Fuentes** [comment]
**issue (blocking):** There is an abrupt skip / jank when the marquee runs out of text. See video below.

https://user-images.githubusercontent.com/43380153/157456370-7ea9573a-95ae-4fb1-ba0f-0755fbfee4e1.mp4

**Comment by Codigo-Fuentes** [comment]
**todo (blocking):** Fix case, formatting of number in stat cards. 

The case of the text between square brackets is inconsistent between the two cards. I suggest using title case--i.e., [Week over week change] and [Total deals]. I think this is most in keeping with overall style on the site.

The number of total deals should probably have a comma (e.g., 34,959) or use toLocaleString.

copy-issues

**Review by Codigo-Fuentes — CHANGES_REQUESTED** [review]
**issue (blocking):** Still getting jank on page load.

https://user-images.githubusercontent.com/43380153/157508972-a83fb10e-7d21-4207-bebc-bf7313705517.mp4


https://user-images.githubusercontent.com/43380153/157508981-047464cf-7055-4eac-b677-52925a7a700e.mp4

**Comment by Codigo-Fuentes** [comment]
**todo:** Fix 1px white strips on sides of image with 3 arrows at top of card. Appears on Safari dt and mobile

1px-white

1px-white-dt

**Comment by JeffLowe** [comment]
Checked on device lab (iphone 8 safari & Android Chrome latest, as well as chrome desktop).  stats page looks good

File: packages/client/test/lib.spec.js [hunk 1]
```diff
@@ -137,6 +137,7 @@ describe('client', () => {
     })
 
     it('upload large CAR with a CarReader', async function () {
+      // @ts-ignore
```
Comment (dashcraft) [inline packages/client/test/lib.spec.js:140]: This blows up the linter so we had to add it here to get the build to pass. It's in a spec file, so there are no negative downstream side effects we need to worry about.

**Review by dashcraft — APPROVED** [review]
Looking good! Good starting off point!

**Pull request merged**